pandas/tests/frame/test_timezones.py

```diff
@@ -166,3 +166,19 @@ def test_constructor_data_aware_dtype_naive(self, tz_aware_fixture):
         result = DataFrame({"d": [pd.Timestamp("2019", tz=tz)]}, dtype="datetime64[ns]")
         expected = DataFrame({"d": [pd.Timestamp("2019")]})
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "initial",
+        ["2018-10-08 13:36:45+00:00", "2018-10-08 13:36:45+03:00"],  # Non-UTC timezone
+    )
+    @pytest.mark.parametrize("method", [DataFrame.min, DataFrame.max])
+    def test_preserve_timezone(self, initial: str, method):
+        # GH 28552
+        # Given
+        initial_dt = pd.to_datetime(initial)
+        series = pd.Series([initial_dt])
+        df = DataFrame([series])
+        # When
+        actual_df = method(df, axis=1)
+        # Then
+        assert actual_df[0].tz == initial_dt.tz
```

Comment:
Could you rename the results `result` and `expected` and compare 2 DataFrame or Series objects and use `tm.assert_equal(result, expected)`

Reply:
Addressed. Please note that `min` and `max` methods of the `DataFrame` both return `Series`, so I choose to use the `tm.assert_series_equal` instead in the same manner as it is being used in `test_frame_no_datetime64_dtype`.